modules/programs/broot.nix

```diff
@@ -0,0 +1,256 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.broot;
+
+  configFile = config:
+    pkgs.runCommand "conf.toml"
+      {
+         buildInputs = [ pkgs.remarshal ];
+         preferLocalBuild = true;
+         allowSubstitutes = false;
+      }
+      ''
+        remarshal -if json -of toml \
+          < ${pkgs.writeText "verbs.json" (builtins.toJSON config)} \
+          > $out
+      '';
+
+  brootConf = {
+    verbs = cfg.verbs;
+    skin = cfg.skin;
+  };
+
+in
+
+{
+  meta.maintainers = [ maintainers.aheaume ];
+
+  options.programs.broot = {
+    enable = mkEnableOption ''
+      A better way to navigate directories.
```

Comment:
The [definition of `mkEnableOption`](https://github.com/NixOS/nixpkgs/blob/a921f8a1d1da72277ec77c6a0195fb5cb098cdf0/lib/options.nix#L63-L70) is
```nix
  mkEnableOption =
    # Name for the created option
    name: mkOption {
    default = false;
    example = true;
    description = "Whether to enable ${name}.";
    type = lib.types.bool;
  };
```
so the argument need to fit the description. In this case it would become

> Whether to enable A better way to navigate directories..

Which becomes a bit odd. I would suggest having something like

```
mkEnableOption "Broot, a better way to navigate directories"
```

to have

> Whether to enable Broot, a better way to navigate directories.

in the documentation 🙂